.github/workflows/main.yml

```diff
@@ -0,0 +1,39 @@
+on:
+  pull_request:
+    types: [opened, reopened, ready_for_review, review_requested]
+  issue_comment:
+    types: [created]
+
+jobs:
+  code_review_job:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+      contents: write
+    name: Run code review agent on every pull request, respond to user comments
+
+    steps:
+      - name: Set Options Environment Variable for Issue Comment
+        if: github.event_name == 'issue_comment'
+        run: |
```

Comment:
**Suggestion**: Avoid hardcoding secrets and paths directly in the workflow file. Use GitHub secrets and environment variables to manage sensitive information and configurations more securely and flexibly. <br> **Code Suggestion**: 
 ```
 - echo "OPTIONS=--static_analysis.fb_infer.enabled=True --code_feedback=True --dependency_check.enabled=False --bee.path=$BEE_PATH  --bee.actn_dir=$BEE_ACTN_DIR --git.access_token=$GIT_ACCESS_TOKEN --bito_cli.bito.access_key=$BITO_ACCESS_KEY $GIT_DOMAIN_OPTION" >> $GITHUB_ENV
 ```